readthedocs/core/management/commands/update_docker.py

```diff
@@ -20,4 +20,6 @@ def handle(self, *args, **options):
                 project_data = api.project(slug).get()
                 p = tasks.make_api_project(project_data)
                 log.info("Building %s" % p)
-                tasks.update_docs(pk=p.pk, docker=True)
+                # TODO just move this to update_api.py
```

Comment:
Also, maybe we consider implementing them as admin functions instead. Cases like build docs for a version from the admin are more useful for rtd.com, but allowing non ops folks to trigger some of the management functionality from the admin might be a win